lib/src/modules/preview/previewImage.scss

```diff
@@ -6,4 +6,18 @@
 
 .preview-image-fallback {
   padding: 10%;
-}
\ No newline at end of file
+}
+
+.preview-children {
```

Comment:
💚 One thing to note is that this change means that the only thing that can be passed as children to `<PreviewImage />` is now `<Controls />`, as the styles applied here are specific to controls.

This is not really a problem, but maybe it could be reflected in the API - instead of `<PreviewImage />` rendering  `children`, it could be passed a `controls` component prop. That way we're telling people "hey, this component expects controls and nothing else!"

Reply:
I understand what you're saying here but in truth, anything can be added here and it'll be positioned correctly.